source/_integrations/wled.markdown

```diff
@@ -126,6 +126,10 @@ Keep Master Light:
   description: Keep the master light, even if there is only 1 segment. This ensures the master light is always there, in case you are automating segments to appear and remove dynamically.
 {% endconfiguration_basic %}
 
+## Communication details / Firewall requirements
+
+The communication between Home Assistant and WLED uses port 80/tcp (tested with WLED 0.14.0). Home Assistant initiates the connection (source port is random).
```

Comment:
This is not Home Assistant-specific information.

It does not help the average user, and user that use these things should be aware (otherwise, they have many more issues, as this is basic HTTP communications).

If any place, this would fit in the WLED documentation, not Home Assistant itself.

../Frenck